Week 6 Assignment Michelle/app.js

```diff
@@ -0,0 +1,29 @@
+// Enable document.ready
+$( document ).ready(function(){
+//Define arrays and options
+	var select = document.getElementById("city-type");
```

Comment:
You can use jQuery here. ```$('#city-type')```. This will also give you the list of jQuery methods too.